docs/get_started/product_overview/api_access.md

```diff
@@ -6,30 +6,30 @@ tags:
+| [**HCL Ring API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/ring-api) | Presents developer entry points to HCL DX 9.5 core platform capabilities, which are divided into two main categories: Authorization APIs and Web Content APIs. | /dx/api/core/v1/explorer |
 | [**HCL Digital Asset Management API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/dam-api) | Presents developer entry points to the Digital Asset Management features of HCL DX 9.5. | /dx/api/dam/v1/explorer |
 | [**HCL Image Processor API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/image-processor-api) | Presents developer entry points to the Digital Asset Management Image Processor functions of HCL DX 9.5. | /dx/api/image-processor/v1/explorer |
 | [**HCL Personalization API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/pzn-api) | Presents developer entry points to the Personalization Rule functions of HCL DX 9.5. | /dx/api/pzn/v1/explorer |
-| [**HCL WCM API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/wcm-api) | Presents redesigned developer entry points to the Web Content Manager functions of HCL DX 9.5. Intended to facilitate the management of Web content, content-sites, and content-pages. | /dx/api/wcm/v2/explorer |
+| [**HCL WCM API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/wcm-api) | Presents redesigned developer entry points to the Web Content Manager functions of HCL DX 9.5. This API is designed to facilitate the management of web content, content-sites, and content-pages. | /dx/api/wcm/v2/explorer |
 | [**HCL Access Control API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/access-api) | Presents developer entry points to the Access Control Management functions of HCL DX 9.5. | /dx/api/access/v1/explorer |
 | [**HCL Remote Model API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/remotemodel-api) | Presents developer entry points to the various models for pages, layouts, and more functions of HCL DX 9.5. | /dx/api/remotemodel/v1/explorer |
 | [**HCL Search API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/search-api) | Presents redesigned developer entry points to the search (like text or keyword searches) functions of HCL DX 9.5. | /dx/api/search/v1/explorer |
 | [**HCL Users and Groups API**](https://HCL-TECH-SOFTWARE.github.io/experience-api-documentation/users-api) | Presents developer entry points to the user and group management functions of HCL DX 9.5. | /dx/api/users/v1/explorer |
 
-Various server-side APIs are also available for providing extensions to the system. Typically, these are Java based APIs.
+Various server-side APIs are also available that provide extensions to the system. Typically, these APIs are Java based.
```

Comment:
```suggestion
Various server-side APIs are also available that provide extensions to the system. Typically, these APIs are Java-based.
```

Reply:
Well, Java is a trademark. Trademarks should not be included in hyphenated phrases.